static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js

```diff
@@ -66,19 +66,6 @@ export const newsletterEmbeds = {
                     });
                 }
             },
-        },
-        {
-            id: 'control',
-            test: () => {
-                const iframes = ((document.querySelectorAll('.email-sub__iframe')));
-                if (iframes) {
-                    iframes.forEach((ifrm) => {
-                        if (ifrm.id !== 'footer__email-form') {
-                            trackComponentInOphan(ifrm.id, 'control');
```

Comment:
We're still going to need tracking though aren't we? or are we going to baseline it?

Reply:
The intention is to not implement the control at all so I don't _think_ we need tracking here? I might be wrong but I think we're just going to compare against pre-test and see if we see a change

Reply:
I'm confused then, isn't there a missing piece that would force the new design to be displayed once the JS loads on the page?

Reply:
I think that should just be the fact that there's only one variant available in the test. If we roll out the test to 100% of the audience and only have one variant, everyone should be put in the variant group and the test logic should run as normal. This should mean that anyone on desktop/browser sees the new embeds but apps aren't affected as they fall back to the default of the existing embed

Reply:
ha off course! I got thrown off by the diff, but expanding the diff there's still the variant above, hence the confusion 👍